hexkit/protocols/objstorage.py

```diff
@@ -198,6 +198,16 @@ async def get_object_download_url(
             bucket_id=bucket_id, object_id=object_id, expires_after=expires_after
         )
 
+    async def get_object_metadata(
+        self, *, bucket_id: str, object_id: str
+    ) -> dict[str, Any]:
```

Comment:
I like this idea, however, the return type should define which data to expect. Or if we currently only want to query the file size, we can also simply name it `get_object_size`. (Or if we currently don't need it as part of the public interface then the _get_object_metadata method is enough, too.) If we keep it as public method, we should also add tests.

Reply:
I'll remove the public-facing part and keep the internals, might need them somewhere else.
Regarding the return type: That's what we get back from the head_object call, so currently I'm just passing it through.
Should we wrap this somehow?